src/components/Input.js

```diff
@@ -3,7 +3,12 @@ import PropTypes from "prop-types";
 import styled from "styled-components";
 
 const SContainer = styled.div`
-  border-bottom: 1px ${props => (props.disabled ? "dashed" : "solid")} #c6c6c6;
+  border-bottom: 1px
```

Comment:
This is clean, but a good place to use new styled-components `css` helper for prop based styling if you want. https://www.styled-components.com/docs/api#css